Add a blog post about github org (#1309)
wakujs/waku: packages/website/private/contents/post-008.mdx

Today, we're excited to announce that Waku has officially moved to a GitHub organization: [https://github.com/wakujs](https://github.com/wakujs).

## A Milestone Toward Openness

This move marks an important milestone. Until now, Waku had been hosted under my personal GitHub account, but we wanted to explicitly signal that this is no longer just a personal project. This move makes it clear that it's a team effort.

Technically, nothing changes in how we collaborate. However, this shift represents our commitment to making Waku an open and community-driven project. We're now more ready than ever to welcome contributors, receive feedback, and shape Waku together.

## What's Next?

We are currently finalizing **v0.22**, which will bring several improvements. Moving to a GitHub organization is just one step in the bigger picture of making Waku more usable and accessible to a wider audience.

## How You Can Get Involved

We'd love for you to be part of this journey. Here's how you can get started:

> - Read the [docs](https://waku.gg) and create a small app

Reviewer: ```suggestion
- Read the [docs](https://waku.gg) and create an app!
```
nit, but we've seen success with larger apps like wes's, so "small" feels unnecessary.

- Experiment with your favorite libraries
- Try converting an existing app to Waku

> - Join discussions, share feedback, and contribute to the repo

Reviewer: maybe a link to the discord server here?

Author: ```suggestion
- Share your feedback in our [GitHub discussions](https://github.com/wakujs/waku/discussions) and in our [Discord server](https://discord.gg/MrQdmzd)
```

This is an exciting time for Waku, and we look forward to growing the community together.

> Looking forward to having more contributors!

Reviewer: ```suggestion
Looking forward to having more contributors and users!
```
maybe?
